contrib/systemd/containerized/origin-master.service

```diff
@@ -2,6 +2,8 @@
 After=docker.service
 Before=origin-node.service
 Requires=docker.service
+After=etcd.service
+Wants=etcd.service
```

Comment:
I'm not sure if we should use `etcd` or `etcd_container` here. We name it `etcd_container` in ansible because we didn't want to uninstall etcd on atomic host but we also didn't want to use it but that doesn't mean that's what we have to do here.